crates/json/src/schema/intern.rs

```diff
@@ -49,8 +48,8 @@ impl Table {
     /// is zero-valued.
     pub fn lookup(&self, s: &str) -> Set {
         match self.m.get(s) {
-            Some(&v) => v,
-            None => 0,
+            Some(v) => v.clone(),
```

Comment:
Based on a quick look at the usages of `lookup`, it seems like this could be modified to return a borrowed `BitSlice` instead of cloning.